linode/manager
# upcoming: [M3-8209] - Region label updates

## Description 📝
When the Gecko GA flag is true:
- Display Country, City instead of City, State anywhere there is a region label
- Ensure Linode Create v2 displays Country, City

I decided to keep the region label updates feature flagged for now since it is a bit of an update, LA/GA has been delayed further, and to avoid confusion with the patching csv list, etc.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/f5fb9d3f-789d-47e4-aeb7-a54db0e1d934) | ![image](https://github.com/user-attachments/assets/3e3fc7c1-d937-432a-9344-426d94990acf) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Ensure your account has the `new-dc-testing`, `edge_testing` and `edge_compute` customer tags

### Verification steps
(How to verify changes)
- Check region labels throughout the app
- Regions should display as Country, City with the gecko flag on and as normal with the flag off

## As an Author I have considered 🤔

*Check all that apply*
